guides/common/modules/proc_deploying_an_application_instance.adoc

```diff
@@ -0,0 +1,54 @@
+[id="{context}_deploying_an_application_instance"]
+= Deploying an Application Instance
+
+You can deploy multi-host applications using xref:{context}_application_instances[application instances].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Applications > App Instances*.
+. Select the application instance you want to deploy and click the *Deploy* button.
+If the selected application instance has been deployed before, you are prompted to confirm redeploying to existing hosts.
+Confirming this automatically starts provisioning the specified hosts and configuring them according to the Ansible playbook.
+
+[CAUTION]
+====
+The firewall settings must be handled by the Ansible playbook itself.
+Make sure to open any required network ports for any defined service by creating an additional role or task in the Ansible playbook.
+====
+
+[id="{context}_deploying_an_application_instance_run_playbook"]
+== Run Playbook
+
+You can rerun the Ansible playbook to reconfigure existing hosts.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Applications > App Instances*.
+. Select the application instance you want to deploy and click the *Run Playbook* button from the drop down menu.
+This will automatically start configuring the specified hosts according to the Ansible playbook.
+
+[id="{context}_deploying_an_application_instance_run_playbook_customize_first"]
+== Run Playbook - Customize First
+
+You can customize and rerun the Ansible playbook to reconfigure existing hosts.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Applications > App Instances*.
+. Select the application instance you want to deploy and click the *Run Playbook - customize first* button from the drop down menu.
+This opens the *Job Invocation* page with prefilled instructions to deploy the application instance.
```

Comment:
```suggestion

```
Trust the user and the UX team here. This wastes their time reading. It's anti-minimalist.

Reply:
Removed.